assets/training/model_management/components/convert_model_to_mlflow/spec.yaml

```diff
@@ -43,6 +43,7 @@ inputs:
     enum:
       - transformers
       - mmlab_pyfunc
+      - stable_diffusion_pyfunc
```

Comment:
nit: why can't we have a single flavor `pyfunc` and decide internally in the source which one to call based on task type? Also asking since @PhaniShekhar is also intending the same in PR: #1180